client/src/components/MyAccount/ProfileTab.jsx

```diff
@@ -258,28 +311,153 @@ const ProfileTab = () => {
+                value={cardDetails.cardNumber}
+              />
+            </div>
+
+            <div className="flex space-x-6 mb-8 mt-4">
+              <div className="flex flex-col flex-1">
+                <label className="text-lg leading-7 font-normal mb-4">
+                  Expiration Date
+                </label>
+                <Select
+                  options={generateCardExpiryYears()}
+                  placeholder="YYYY"
+                  styles={customStyles}
+                  isSearchable={false}
+                  components={{
+                    IndicatorSeparator: () => null,
+                  }}
+                  onChange={(selectedOption) => {
+                    setCardDetails({
+                      ...cardDetails,
+                      expirationYear: selectedOption.value,
+                    });
+                  }}
+                />
+              </div>
+
+              <div className="flex flex-col flex-1 justify-end">
+                <Select
+                  options={generateMonths()}
+                  placeholder="MM"
+                  styles={customStyles}
+                  isSearchable={false}
+                  components={{
+                    IndicatorSeparator: () => null,
+                  }}
+                  onChange={(selectedOption) => {
+                    setCardDetails({
+                      ...cardDetails,
+                      expirationMonth: selectedOption.value,
+                    });
+                  }}
+                />
+              </div>
+
+              <div className="flex flex-col flex-1">
+                <label className="text-lg leading-7 font-normal  mb-4">
+                  CVC/CVV
+                </label>
+                <div className="border-2 h-12">
+                  <input
+                    type="password"
+                    maxLength={4}
+                    className="w-full h-full outline-none px-6 bg-bgWhite"
+                    placeholder="***"
+                    onChange={(e) => {
+                      if (e.target.value.match("^[0-9]*$") != null) {
+                        setCardDetails({
+                          ...cardDetails,
+                          cvc: e.target.value,
+                        });
+                      }
+                    }}
+                    value={cardDetails.cvc}
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/*Shipping address form */}
       <div className="w-full border-2 mt-6 font-normal">
         <h2
           className="px-8 py-4 text-lg font-normal leading-7 bg-bgWhite cursor-pointer flex items-center"
-          onClick={toggleTabOpened}
+          onClick={toggleShippingTabOpened}
         >
           <span className="mr-4">
-            {isOpened ? <BsChevronUp /> : <BsChevronDown />}
+            {isShippingTabOpened ? <BsChevronUp /> : <BsChevronDown />}
```

Comment:
This part is now repeating for card as well, we could create <Tab> component that would handle this code for displaying and hiding content.